src/libappimage/desktop_integration/Thumbnailer.cpp

```diff
@@ -102,10 +103,11 @@ namespace appimage {
                 /* thumbnails are always png */
                 iconHandle.save(largeThumbnailPath.string(), "png");
                 return;
-            } catch (const IconHandleError&) {
+            } catch (const IconHandleError& error) {
                 /* we fail to resize the icon because it's in an unknown format or some other reason
                  * we just have left to write it down unchanged and hope for the best. */
-                Logger::warning("Unable to resize the application icon into a 256x256 image, "
+                Logger::warning(error.what());
```

Comment:
Would prefer one warning instead of two. Right now the messages appear unrelated to each other. I guess appending `what()` should do.

Reply:
Fixed, example below:
`WARNING: Unable to resize the application icon into a 128x128 image: "png resizing is not supported". It will be written unchanged.`

Reply:
Isn't this a little dangerous? Shouldn't there be some "static" text as well?

I mean, the message looks fine.

Reply:
The two messages were mixed, please check the new changes, the ones shown above are outdated.